operator_ui/src/hooks/useLoadingPlaceholder.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+
+export const useLoadingPlaceholder = (
```

Comment:
What do you think about using useState and useEffect on isLoading and LoadingPlaceholder. Would that be perf improvement?